docs/production/authentication-methods.md

```diff
@@ -655,11 +657,24 @@ another IdP.
    /home/zulip/deployments/current/manage.py logout_all_users
    ```
 
+#### SP-initiated Logout
+
+After configuring IdP-initiated Logout, you only need to set
+`SAML_ENABLE_SP_INITIATED_SINGLE_LOGOUT = True` in `/etc/zulip/settings.py`
+to also enable SP-initiated Logout. When this is active,
+a user who logged in to Zulip via SAML, upon clicking
+"Logout". will be redirected to the IdP's SLO endpoint with a LogoutRequest
```

Comment:
`"Logout",`, I assume?

Should we expand "SLO"?

Reply:
Made both tweaks